rocAL/include/pipeline/commons.h

```diff
@@ -154,3 +154,15 @@ struct Timing {
     long long unsigned video_decode_time= 0;
     long long unsigned video_process_time= 0;
 };
+
+/*! \brief Tensor Last Batch Policies
+ These policies the last batch policies determine the behavior when there are not enough samples in the epoch to fill the last batch
+        FILL - The last batch is filled by either repeating the last sample or by wrapping up the data set.
+        DROP - The last batch is dropped if when there are not enough samples from the current epoch.
```

Comment:
Is it in an epoch? Or in a shard?

Reply:
The wordings are taken from DALI as is

Reply:
Please check with Shobana once if we need to exactly have the description from DALI
Though we replicate their functionality better to have description in our own terms. Please ensure this in subsequent PR's